idea-plugin/src/com/reason/bs/BsLineProcessor.java

```diff
@@ -153,12 +156,15 @@ private OutputInfo extractExtendedFilePositions(@Nullable String text) {
   }
 
   // ...path/src/Source.re 111:21-112:22
+  // ...path/src/Source.re:111:21-112:22
   // ...path/src/Source.re 111:21-22
+  // ...path/src/Source.re:111:21-22
   // ...path/src/Source.re 111:21   <- must add 1 to colEnd
+  // ...path/src/Source.re:111:21   <- must add 1 to colEnd
```

Comment:
comments are duplicated

Reply:
there's a slight difference. the new version of bs-platform uses a colon instead of a space between file names and line numbers

Reply:
oh right, didnt see it